src/main/java/jesseg/ibmi/opensource/nginx/NginxConfNode.java

```diff
@@ -39,9 +41,18 @@ public static NginxConfNode open(final File _file) throws IOException {
                     curStr = "";
                 } else if (';' == c) {
                     curStr = curStr.trim();
-                    final String property = curStr.replaceAll("\\s.*", "");
-                    final String value = curStr.replaceFirst("^[^\\s]*\\s+", "");
-                    currentNode.addProperty(property, value);
+                    Pattern pattern = Pattern.compile("^([^\\s]+)\\s*($|.*$)"); //"^(\\S*)\\s*($|.*$)"
+                    Matcher matcher = pattern.matcher(curStr);
+                    String property = null;
+                    String value = null;
+                    while (matcher.find()) {
+                        property = matcher.group(1);
+                        value = matcher.group(2);
+                    }
+                    System.out.println("PARSE CONFIG  >>>>>>>>>>>>>>>>>>>>>");
```

Comment:
It's probably a good idea to have some kind of debug output. We should leverage the `AppLogger` class and its various `_verbose` functions (see other examples throughout the code). That way, we will get this information if the user invokes with `-v`. 

That means we'll need to refactor the constructor and factory methods at minimum, and change the caller to pass in an `AppLogger` instance.